inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -685,20 +686,43 @@ linelist_cleaned$age_group <- age_categories(linelist_cleaned$age_years,
 #                                            "15-29 years",
 #                                            "30-44 years", 
 #                                            "45+ years")))
+```
+
+
+
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+create_vars
+
+This chunk will help you construct new variables from other variables.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+```{r create_vars}
+
+## Numeric variables -----------------------------------------------------------
+# This step creates a variable for number of days under observation.
+# You can adapt this step to create other calculated variables
+
+# create number of days under observation
+linelist_cleaned <- mutate(linelist_cleaned, 
+                           obs_days = as.numeric(date_of_exit - 
+                                                   date_of_consultation_admission))
+
+
+
+# Factor (categorical) variables -----------------------------------------------
 
-# Factor (categorical) variables ---------------------------------
+# This step creates a variable from another character/factor variable
+# You can adapt this step to create other calculated variables
 
-## create a new variable from another character/factor variable
-## create a binary variable if patients died 
-## use this version if you have named values 
-## (e.g. "Dead on arrival" or "Dead in facility")
-## alternatively use the version below
+# The variable DIED will be binary -- if patient died or not
+# if your variable is named ("Dead on arrival", "Dead in facility"), use this
 linelist_cleaned$DIED <- grepl("Dead", linelist_cleaned$exit_status)
```

Comment:
I'm thinking if it would be better to change `grepl("thing", data$variable)` to `str_detect(data$variable, "thing")` to better align with the whole `data$variable %in% c("thing", "thang")` vibe.